lib/Hashids.pm

```diff
@@ -33,13 +33,13 @@ has alphabet => (
 
 has chars => ( is => 'rwp', init_arg => undef, default => sub { [] } );
 
-has seps => (
+has separators => (
```

Comment:
I'm fine with using "seps" rather than the longer "separators" keyword.  I also like your suggestion for this preference because, now that I think of it, it can prevent problems resulting from a somewhat common misspelling that comes out as "seperators."